lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,450 @@
+import net from 'net';
+import fs from 'fs';
+import cluster from 'cluster';
+import events from 'events';
+import assert from 'assert';
+
+import level from 'level';
+import multilevel from 'multilevel';
+import sublevel from 'level-sublevel';
+import arsenal from 'arsenal';
+import { logger } from '../../utilities/logger';
+
+import { errors } from 'arsenal';
+import BucketInfo from '../BucketInfo';
+import constants from '../../../constants';
+
+const METADATA_PORT = 9990;
+const METADATA_PATH = '/metadata/';
```

Comment:
Please use:
const METADATA_PATH = config.filePaths.metadataPath;

This will be available in the data backend PR.